docs/resources.md

```diff
@@ -0,0 +1,215 @@
+# Resources
+
+## Computable Resources
+
+KUDO Cassandra by default requests 1.5 cpu and 4.5Gi of memory for each
+Cassandra node. The limits by default are 2 cpu and 4.5Gi of memory.
+
+Those requests and limits can be tuned using the parameters:
+
+- NODE_CPU_MC
+- NODE_CPU_LIMIT_MC
+- NODE_MEM_MIB
+- NODE_MEM_LIMIT_MIB
+- PROMETHEUS_EXPORTER_CPU_MC
+- PROMETHEUS_EXPORTER_CPU_LIMIT_MC
+- PROMETHEUS_EXPORTER_MEM_MIB
+- PROMETHEUS_EXPORTER_MEM_LIMIT_MIB
+- RECOVERY_CONTROLLER_CPU_MC
+- RECOVERY_CONTROLLER_CPU_LIMIT_MC
+- RECOVERY_CONTROLLER_MEM_MIB
+- RECOVERY_CONTROLLER_MEM_LIMIT_MIB
+
+## Storage resources
+
+By default KUDO Cassandra uses 20GiB PV. This isn't recommended for production
+use. Please refer to [production](./production.md) docs to see the storage and
+compute resources recommendations
+
+## Resources per container
+
+#### Cassandra container
+
+```
+resources:
+  limits:
+    cpu: 1
+    memory: 4Gi
+  requests:
+    cpu: 1
+    memory: 4Gi
+```
+
+#### Bootstrap init container
+
+```
+resources:
+  limits:
+    cpu: 200m
+    memory: 256Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+```
+
+#### prometheus exporter sidecar
+
+```
+resources:
+  limits:
+    cpu: 1
+    memory: 512Mi
+  requests:
+    cpu: 500m
+    memory: 512Mi
+```
+
+#### cassandra-recovery controller pod
+
+```
+resources:
+  limits:
+    cpu: 200m
+    memory: 256Mi
+  requests:
+    cpu: 50m
+    memory: 50Mi
+```
+
+## Kubernetes Objects
+
+KUDO Cassandra is delivered with a different set of features, enabling or
```

Comment:
`different` from what? Are you trying to say that it's delivered with a _specific_ set of features, which can be disabled if needed?

Reply:
right, but by default, those are disabled, and enabling those would create more kubernetes objects than default installation. Let me try to reword that